security/cert_manager_operator/cert-manager-nw-policy.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-nw-policy"]
+= Network policy configuration for operands
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-nw-policy
+
+toc::[]
+
+The {cert-manager-operator} provides predefined `NetworkPolicy` resources to enhance security by controlling ingress and egress traffic for its components. By default, this feature is disabled to prevent connectivity issues or breaking changes during an upgrade. To use this feature, you must enable it in the CertManager custom resource (CR). After enabling the default policies, you must manually configure additional egress rules to allow {cert-manager-operator} components to communicate with any external issuer providers.
```

Comment:
```suggestion
The {cert-manager-operator} provides predefined `NetworkPolicy` resources to enhance security by controlling ingress and egress traffic for its components. By default, this feature is disabled to prevent connectivity issues or breaking changes during an upgrade. To use this feature, you must enable it in the CertManager custom resource (CR).

After enabling the default policies, you must manually configure additional egress rules that allow outbound traffic required for your certificate issuers, such as ACME servers (Let's Encrypt), DNS-01 challenge providers (Route53, CloudFlare), Vault, or any other external CAs. Without additional egress rules, cert-manager cannot reach external services beyond the API server and internal DNS.
```